contrib/clojure-package/doc/intro.md

```diff
@@ -4,9 +4,8 @@ A clojure package to the MXNet Deep Learning library
 
 ## Introduction
 
-MXNet is a first class, modern deep learning library that AWS has officially picked as its chosen library. It supports multiple languages on a first class basis and is incubating as an Apache project.
+MXNet is a first class, modern deep learning library. It supports multiple languages on a first class basis and is incubating as an Apache project.
 
 The motivation for creating a Clojure package is to be able to open the deep learning library to the Clojure ecosystem and build bridges for future development and innovation for the community. It provides all the needed tools including low level and high level apis, dynamic graphs, and things like GAN and natural language support.
```

Comment:
Is this section intended to be rendered on the website or just as part of the package source code?

Reply:
It prints out as part of the packaged api-docs that are generated by `lein codox`. I didn't really know what to say so I just repeated the intro of the README.  I'll remove since it doesn't really shed anything into the api docs